tests/integration/numbers/PhoneNumbersTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+class PhoneNumbersTest extends BaseTest
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->client = new \MessageBird\Client('YOUR_ACCESS_KEY', $this->mockClient);
+    }
+
+    public function testReadPhoneNumber()
```

Comment:
```suggestion
    public function testReadPhoneNumber(): void
```

In the other methods as well